src/components/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface HeaderData {
+  searchText: string;
+  onChangedSearchText: Function;
```

Comment:
Usually, callback's types are defined more specifically. E.g. for this case(if this callback is useState setter) you can use `React.Dispatch<React.SetStateAction<string>>`

Reply:
Fixed.